### ISSUE TITLE
(cosmetics) persistent cmd window after "writing default profile"

### ISSUE THREAD

**Issue by LigH-de** [jb-alvarado/media-autobuild_suite#876]
After an MSYS2 update where the last message in the cmd window for media-autobuild_suite.bat is:
```
-------------------------------------------------------------------------------
writing default profile
-------------------------------------------------------------------------------
```
this window does not disappear after mintty was launched, even persists after the suite finishes.

**Comment by BlackScreen** [comment]
Had this also from time to time. I just closed the window. But yeah, a fix would be better.

**Comment by Barough** [comment]
I have this 'issue' also.

**Comment by LigH-de** [comment]
This is now regular again. The default profile is written every run.

**Comment by schmidthubert** [comment]
Ask Alexpux ?

**Comment by LigH-de** [comment]
I believe wiiaboo recently added keyring management. So it may be related to checking the signatures.

**Comment by schmidthubert** [comment]
All he did was adding his own repo ...

**Comment by schmidthubert** [comment]
Didn't you have that window problem weeks ago already? That was well before the addition of that repo.

**Comment by LigH-de** [comment]
Yes, it happened before. Only when a default profile was written. That happened only rarely, only during an MSYS2 update which has to be applied in the cmd window, not in a mintty shell.

But for a little while now, `writing default profile` appears every time the suite runs. There must be a reason ...

**Comment by LigH-de** [comment]
Via IRC (#msys2 on OFTC), 'dwfreed' suggested to use `gpg-connect-agent killagent /bye` after updating the key ring.

**Comment by LigH-de** [comment]
I hope @Alexpux has at least a clue about the mechanics: https://github.com/Alexpux/MSYS2-pacman/issues/56
